DuckDuckGo.Fluent.Plugin/JsonResult.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+
+namespace DuckDuckGo.Fluent.Plugin
+{
+    public record JsonResult
+    {
+        public record Icon
+        {
+            public string Url { get; set; }
+        }
+
+        public class Topic
+        {
+            public string FirstUrl { get; set; }
+            public Icon Icon { get; set; }
+            public string Text { get; set; }
+        }
+
+        public record RelatedTopic
+        {
+            public string FirstUrl { get; set; }
+            public Icon Icon { get; set; }
+            public string Text { get; set; }
+            public string Name { get; set; }
+            public List<Topic> Topics { get; set; }
+        }
+
+
+        public record Root
```

Comment:
Rename to something meaningful, like `DuckDuckGoApiResult` or something like that